command/windows_service.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package command
+
+import (
+	"strings"
+
+	"github.com/hashicorp/cli"
+)
+
+type WindowsServiceCommand struct {
+	Meta
+}
+
+func (c *WindowsServiceCommand) Help() string {
+	helpText := `
+Usage: nomad windows service <subcommand> [options]
+
+  This command groups subcommands for managing nomad as a system service on Windows.
+
+  Install:
+
+      $ nomad windows service install
+
+  Uninstall:
+
+      $ nomad windows service uninstall
+
+  Please see the individual subcommand help for detailed usage information.
```

Comment:
```suggestion
  Refer to the individual subcommand help for detailed usage information.
```

"See" is considered ableist language, so we use "Refer to" instead. CLI/API output should follow the same style guide rules as the docs, although I admit I haven't been checking most of the time.